From 1ede9e2d077404ce6f78676ad68d20244282860f Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Tue, 22 Mar 2022 01:02:14 +0100
Subject: [PATCH 453/456] arm64: dts: rk3399-pinephone-pro: Fix Alt-DP VDO pin
 assignements

Since Pinephone has a DFP_D supporting receptacle, we need to set
DFP_D pin assignments via bits 15:8 and not via bits 23:16 which
are for UFP_D.

Previous VDO worked, because displayport.c driver was broken. With
the driver fixed, proper VDO can be used and docks with Type-C
receptacles will work with Pinephone Pro.

See also: https://xnux.eu/log/#063

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 450499347c3d..219bd80e254b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -1013,7 +1013,7 @@ connector {
 			altmodes {
 				dp {
 					svid = <0xff01>;
-					vdo = <0x0c0046>;
+					vdo = <0x0c46>;
 				};
 			};
 
-- 
2.34.1

